bitnami/kube-prometheus/Chart.yaml

```
@@ -34,4 +34,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-prometheus
   - https://github.com/bitnami/bitnami-docker-alertmanager
   - https://github.com/prometheus-operator/kube-prometheus
-version: 3.6.0
+version: 3.6.1
```

Please bump the major version instead, since you're including changes that break backwards compatibility:
```suggestion
version: 4.0.0
```

---

You will also have to update the "Upgrading" section of the README.md file explaining why we introduced this major version. Check how @javsalgar did it at https://github.com/bitnami/charts/pull/5010

---

Done.
I added the `tls-secrets.yaml` manifest, bump the chart version to the next major version and updated `README.md` with explanation.